core/src/main/scala/com/stripe/dagon/Dag.scala

```diff
@@ -337,6 +337,16 @@ sealed abstract class Dag[N[_]] extends Serializable { self =>
     def toLiteral = node2Literal
   }
 
+  // these are included for binary compatibility
+
+  // $COVERAGE-OFF$
+  def com$stripe$dagon$Dag$$copy$default$2(): com.stripe.dagon.FunctionK[N, Literal[N, ?]] =
+    self.toLiteral
+
+  def com$stripe$dagon$Dag$$copy$default$3(): scala.collection.immutable.Set[Id[_]] =
```

Comment:
I bet if you make these `private[paiges]` it is still binary compatible but no one in scala can see them. What do you think about that?